Automattic/wp-calypso
# Import in Signup: Show the `welcome` inline help dialog on done

Note: Targeting branch in #31817 -- will repoint to master once it's merged

#### Changes proposed in this Pull Request

* New state piece: `(bool) importerNux.isFromSignupFlow`
* Actions & selector for the above
* When truthy, include the `welcome` query arg in the link to the `View Site` section:

![Screen Shot 2019-03-28 at 11 32 20 AM](https://user-images.githubusercontent.com/1587282/55173626-6e3a6680-5152-11e9-85d6-adbb14027c64.png)


#### Testing instructions

* Follow test plan in #31817 (browse to the import section and complete flows)
  * You should **NOT** see the "welcome" inline help panel
* Complete an import signup flow (`/start/import`)
  * Once the import completes, when you click `Done`, you should land at the `View Site` section & the "welcome" inline help panel should show
